spec/Flagbit/Bundle/CategoryBundle/Connector/Processor/Normalization/ProcessorDecoratorSpec.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+declare(strict_types=1);
+
+namespace spec\Flagbit\Bundle\CategoryBundle\Connector\Processor\Normalization;
+
+use Akeneo\Pim\Enrichment\Component\Category\Model\CategoryInterface;
+use Akeneo\Tool\Component\Batch\Item\InvalidItemException;
+use Akeneo\Tool\Component\Connector\Processor\Normalization\Processor;
+use Flagbit\Bundle\CategoryBundle\Connector\ArrayConverter\StandardToFlat\CategoryProperty as StandardToFlatConverter;
+use Flagbit\Bundle\CategoryBundle\Connector\Processor\Normalization\ProcessorDecorator;
+use Flagbit\Bundle\CategoryBundle\Entity\CategoryProperty;
+use Flagbit\Bundle\CategoryBundle\Repository\CategoryPropertyRepository;
+use PhpSpec\ObjectBehavior;
+use Symfony\Component\Serializer\Exception\ExceptionInterface;
+
+/**
+ * @method process($item)
+ */
+class ProcessorDecoratorSpec extends ObjectBehavior
+{
+    public function let(
+        CategoryPropertyRepository $categoryPropertyRepository,
+        StandardToFlatConverter $standardToFlat,
+        Processor $inner
+    ): void {
+        $this->beConstructedWith($categoryPropertyRepository, $standardToFlat, $inner);
+    }
+
+    public function it_is_initializable(): void
+    {
+        $this->shouldHaveType(ProcessorDecorator::class);
+    }
+
+    /**
+     * @throws InvalidItemException
+     * @throws ExceptionInterface
+     */
+    public function it_should_not_merge_properties_if_missing(
+        CategoryInterface $item,
+        CategoryPropertyRepository $categoryPropertyRepository,
+        Processor $inner
+    ): void {
+        $categoryPropertyRepository->findByCategory($item)->willReturn(null);
+
+        $inner->process($item)->willReturn([
+            'code' => 'test',
+            'parent' => 'master',
+            'label' => [
+                'de_DE' => 'Test',
+                'en_EN' => 'Test',
+            ],
+        ]);
+
+        $this->process($item)->shouldBeLike([
```

Comment:
```suggestion
        $this->process($item)->shouldReturn([
```

Because otherwhile `[4 => '']` and `[4 => null]` and similar comparison would be considered the same.